src/Component/Card/Card.jsx

```diff
@@ -1,72 +1,103 @@
-import { FaRegEye, FaShareAlt } from "react-icons/fa";
+import { FaShareAlt, FaStopwatch } from "react-icons/fa";
 import { BsThreeDotsVertical } from "react-icons/bs";
+import { MdPlaylistAdd, MdRemoveCircle } from "react-icons/md";
 import { NavLink } from "react-router-dom";
-import { useVideo } from "../../Context/Video-Context/video-context";
-import { Navigation } from "../Nav/Navbar";
 import "./Card.css";
-const Card = () => {
-  const { videoData } = useVideo();
+import { useWatch } from "../../Context/WatchLater-context/Watch-context";
+import { useState } from "react";
```

Comment:
use absolute path instead of ../../ this

Reply:
this something new for me.

Reply:
I will help with this